components/location-model/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+const Location = ({ inputChange, clickHandler, locationModal }) => {
+  return (
+    <div>
+      {!locationModal ? null : (
```

Comment:
Try an `if` to bail out early or `&&` short-circuiting instead. Think twice if one of your ternary branches does nothing.